data/campaigns/Heir_To_The_Throne/scenarios/13_The_Dwarven_Doors.cfg

```diff
@@ -160,7 +160,78 @@
         #
         {VARIABLE_OP true_entrance_location rand "1..2"}
 
-        # TODO: manually add snow detritus
+        [lua]
+            code=<<
+
+local wml_actions = wesnoth.wml_actions
```

Comment:
This is a fairly large chunk of Lua code. I wonder if it should be moved to a separate file and loaded in with `wesnoth.dofile`? (Maybe it's not important since it's a one-use thing.)